### PR TITLE
German translations and review existing translations

### PR DESCRIPTION
**Pull request by rebplu**

### CODE REVIEW

File: Nie-ontologies/Generic-ontologies/calendar.ttl [hunk 1]
```diff
@@ -260,7 +260,7 @@ calendar:julianListToJDN
 calendar:julianSetListToDate
 	rdfs:isDefinedBy <http://e-editiones.ch/ontology/calendar#>;
 	a owl:DatatypeProperty;
-	rdfs:label "Julian set list to date"@en, "julianische Reihenliste zu Datum"@de;#
+	rdfs:label "set Julian date list to date"@en, "gesetzte julianische Datumliste zu Datum"@de;
```
Comment (hanscools) [inline Nie-ontologies/Generic-ontologies/calendar.ttl:263]: OK

File: Nie-ontologies/Generic-ontologies/creating.ttl [hunk 2]
```diff
@@ -45,21 +45,21 @@ creating:AgentCreator
 creating:PersonCreator
 	rdfs:isDefinedBy <http://e-editiones.ch/ontology/creating#>;
 	a rdfs:Class;
-	rdfs:label "creator role of person"@en, "Erstellerrolle eines Akteur"@de;
+	rdfs:label "creator role of person"@en, "Erstellerrolle eines Akteurs"@de;
```
Comment (hanscools) [inline Nie-ontologies/Generic-ontologies/creating.ttl]: "Erstellerrolle eines Akteurs" -> "Erstellerrolle einer Person"

File: Nie-ontologies/Generic-ontologies/creating.ttl [hunk 3]
```diff
@@ -45,21 +45,21 @@ creating:AgentCreator
...
 	rdfs:label "author role of person"@en, "Autorrolle einer Person"@de; #, "auteur"@fr, "autore"@it;
-	rdfs:comment """Role of a person being responsible for a creation."""@en;#comment ok like this?
+	rdfs:comment """Role of a person being responsible for a creation."""@en;
 	rdfs:subClassOf human:PersonRole.
 
 creating:Creating 
 	rdfs:isDefinedBy <http://e-editiones.ch/ontology/creating#>;
 	a rdfs:Class;
-	rdfs:label "person-creating"@en, "Erstellen"@de; #why not just creating (similar to the next (creation))
+	rdfs:label "creating"@en, "Erstellung"@de; #why not just creating (similar to the next (creation))
```
Comment (hanscools) [inline Nie-ontologies/Generic-ontologies/creating.ttl]: Remove comment.
reb: last commit in creating.ttl did not work. New commit 9eeda18 with all discussed changes.

File: Nie-ontologies/Generic-ontologies/human.ttl [hunk 4]
```diff
@@ -184,28 +184,28 @@ human:AgentUser
 	rdfs:isDefinedBy <http://e-editiones.ch/ontology/human#>;
 	a rdfs:Class;
 #	rdfs:isDefinedBy <http://e-editiones.ch/ontology/human#>;
-	rdfs:label "agent user"@en, "benutzender Akteur"@de;
+	rdfs:label "agent role of user"@en, "Akteurrolle eines Benutzers"@de;
```
Comment (hanscools) [inline Nie-ontologies/Generic-ontologies/human.ttl]: "agent role of user" -> "user role of agent"; idem for the other roles
reb: ups sorry for that...

**Comment by rebplu** [comment]
Shall we try to merge? I will then go over the changed files again briefly. But rather in a new branch. I have no overview anymore after so many commits...:-)

**Review by hanscools — APPROVED** [review]

**Review by hanscools — APPROVED** [review]

**Pull request merged**